File: Back-End/Entidades/Automovilista.cs
namespace Entidades
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.Text;

    public class Automovilista
    {
        public Guid Id { get; set; }
        [StringLength(30)]
        public string Nombre { get; set; }
        [StringLength(50)]
        public string Apellidos { get; set; }
        public byte Puntos { get; set; }

        public List<AutoAutomivilista> AutosAutomivilistas { get; set; }
    }
}

File: Front-End/src/app/Modelos/automovilista.ts
export class Automovilista {
    public id : string;
    public nombre : string;
    public apellidos : string;
    public puntos : number;

    //public List<AutoAutomivilista> AutosAutomivilistas { get; set; }
    public constructor( data: any = {}) {
        this.id = data.id || '' ;
        this.nombre = data.nombre || '' ;
        this.apellidos = data.apellidos || '' ;
        this.puntos = data.puntos || 0 ;
    } 
}
